src-v2/Not/Not.Blazor/TM/NotChildren.razor

```diff
@@ -45,18 +47,25 @@ else
     public string NoItemsMessage { get; set; } = default!;
     [Parameter]
     public string UpdateRoute { get; set; } = default!;
+    [Parameter]
+    public string Class { get; set; } = default!;
 
     public IEnumerable<T> Items { get; set; } = default!;
 
     protected override void OnInitialized()
     {
+        Name = Localizer.Get(Name);
+        if(NoItemsMessage == default)
+        {
+            string defaultMessage = $"No {Name.ToLower()} have been created for this event.";
+            NoItemsMessage = Localizer.Get(defaultMessage);
```

Comment:
When defined like this you will have to add multiple entries in the localization resex files - one for every usage of `NotChildren` with a different `Name`. Instead you could define the property as a constant that contains a string placeholder and use `string.Format`. That way we will have to translate a single resex entry.  However this isn't currently supported by the `Ilocalizer` so you can just leave a TODO comment so we don't forget about it later.